src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -252,7 +264,7 @@ public void writeBody(PrintWriter w) {
                                     LOGGER.info("Scheduling " + project.getFullDisplayName() + " to build commit " + sha1);
                                     scmTriggerItem.scheduleBuild2(scmTriggerItem.getQuietPeriod(),
                                             new CauseAction(new CommitHookCause(sha1)),
-                                            new RevisionParameterAction(sha1));
+                                            new RevisionParameterAction(sha1), new ParametersAction(buildParameters));
```

Comment:
This results in an empty "Parameters" side panel action for builds of jobs that don't even have parameters defined.